src/data/markdown/docs/02 javascript api/02 k6/group- name- fn -.md

```diff
@@ -17,6 +17,19 @@ Run code inside a group. Groups are used to organize results in a test.
 | ---- | ------------------------- |
 | any  | The return value of _fn_. |
 
+<Blockquote mod="warning" title="Working with async functions">
```

Comment:
I didn't realize that we need to downgrade the admonition. This is more of a "proceed carefully", not "you will do irreparable harm".

https://developers.google.com/style/notices

Reply:
To be honest using  async and group works so strange (given the input from other k6 OSS contributors), even with changes that are *not* landing, that it is more or less "if you do this we do not give you any gurantees how it will work". Which kind of borders on "if it will work"

Reply:
But, to quote the style guidelines:

> Stronger than a caution notice; it means "Don't do this" or that this step might be irreversible, such as leading to permanent data loss. If a reader doesn't heed the warning, they can lose money, lose work, or open themselves to a security breach. For example, "Don't put a password on the command line; doing so is a security risk."

I mean, if async is _this bad_ in groups, is it possible to just forbid completely?

Reply:
we do that to the extend that is possible and then this is here to both note that this is the case, link to the issue that explains this in more details and tell people to not even try to circument this as it will likely not work as they think it will.